chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -690,9 +677,12 @@ func (n *NeutrinoNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 	//
 	// We'll update our filter first to ensure we can immediately detect the
 	// spend at tip.
+	if outpoint == nil {
+		outpoint = &chainntnfs.ZeroOutPoint
+	}
```

Comment:
Do this as first thing in this function, so that `txNotifier.RegisterSpend` can accept it as a value parameter?